src/sentry/static/sentry/app/utils/withEnvironment.jsx

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
-import createReactClass from 'create-react-class';
 import Reflux from 'reflux';
+import createReactClass from 'create-react-class';
 
+import {getDisplayName} from 'app/utils/getDisplayName';
```

Comment:
I don't think you should manually modify import order for every touched file if it's not an agreed upon order. Happy to have this in prettier or eslint though.

Reply:
It's from an editor add-on but agreed would be nice to have eslint auto fix this